KIF Tests/SystemAlertTests_ViewTestActor.m

```diff
@@ -8,13 +8,12 @@
 
 
 #import <KIF/KIF.h>
-#import "KIFUITestActor.h"
 
-@interface SystemAlertTests_ViewTestActor : KIFTestCase
+@interface ZSystemAlertTests_ViewTestActor : KIFTestCase
```

Comment:
Change back to `SystemAlertTests_ViewTestACtor`

Reply:
done in https://github.com/openium/KIF/commit/4f286cf6f0264c1b93b5b1f860f00ca0bfb43104